concat.js

```diff
@@ -4,16 +4,22 @@ const concat = (array, newEl) => {
 
         let otherArray = [...newArray, currentEl];
 
-         if(index == array.length - 1) {
-            otherArray = [...otherArray, newEl]; 
-         }
-
+            if(index == array.length - 1) {
+                
+                if(Array.isArray(newEl)) {
+                    otherArray = [...otherArray, ...newEl]; 
+                } else {
+                    otherArray = [...otherArray, newEl]; 
+                } 
+             }
+         
```

Comment:
please simplify this function , you just need to add a new element to the array, or concat two arrays